rust/gui-client/src-tauri/src/client/debug_commands.rs

```diff
@@ -2,13 +2,10 @@
 //! them with the GUI, or to exercise features programmatically.
 
 use anyhow::Result;
-use tracing::subscriber::set_global_default;
-use tracing_subscriber::{fmt, layer::SubscriberExt as _, EnvFilter, Layer, Registry};
 
 #[derive(clap::Subcommand)]
 pub(crate) enum Cmd {
     SetAutostart(SetAutostartArgs),
-    TrayMenu,
 }
```

Comment:
can we not have a subdommand now?

Reply:
This was for debugging a previous PR and I didn't need it anymore